torch/csrc/jit/passes/frozen_ops_to_mkldnn.cpp

```diff
@@ -238,6 +281,17 @@ Operation BroadOp(const Node* node) {
   };
 }
 
+const RegisterOperators MKLDNNHardSwishOpReg({
+    torch::jit::Operator(
```

Comment:
nit: we have `prim::mkldnn_convolution`, reason not to put this in as `prim::hardswish` for consistency ?